src/main.rs

```diff
@@ -12,32 +13,17 @@ use std::{
 };
 
 use crate::config::CrunchyConfiguration;
+use crate::nodes::{create_nodes, Node};
 use serde::{Deserialize, Serialize};
-use spectre::{graph::AGraph, graph::Graph};
-use ziggurat_core_geoip::geoip::GeoInfo;
+use spectre::graph::AGraph;
 use ziggurat_core_geoip::providers::ip2loc::Ip2LocationService;
 use ziggurat_core_geoip::providers::ipgeoloc::{BackendProvider, IpGeolocateService};
 
-//TODO(asmie): there is some redundancy here as we have ip in the Node structure and one more time
-// in the GeoIPInfo structure.
```

Comment:
This should be done - on myside I've changed the structures in zg-core - if @kylegranger is using it (`GeoInfo` instead of `GeoIPInfo` - and I think that he is now :) ) then this comment can be removed as todo is complete.

Reply:
This TODO is definitively done, hence its removal.